docs/project/suggested-wallets.mdx

```diff
@@ -5,27 +5,31 @@ sidebar_label: Suggested Wallets
 slug: /suggested-wallet
 ---
 
-# Suggested Wallets
-
 import Tabs from "@theme/Tabs";
 import useBaseUrl from "@docusaurus/useBaseUrl";
 
-As we are still in testnet, we connecting a **MetaMask** wallet with WeatherXM. That is because by following
-our guide is fairly easy to use the Polygon's testnet on MetaMask and add a custom token (in our case, the WXM).
+While we are on the testnet, we want to use a **MetaMask** wallet with WeatherXM. By following our guide, it's 
```

Comment:
`we want to use a **MetaMask** wallet` -> `we suggest you use a **MetaMask** wallet`